### PR TITLE
Integrate Initial Project Section Structure with Content

### PR DESCRIPTION
**Pull request by samuelsogbesan**
- Update HTML Structure for project section to contain project cards. Each project card contains the project name
- Convert Project Section to CSSGrid for formatting cards. 
- Add Initial Styling for Project Cards including a hover rule.

Note the first 3 commits are pending from another PR (#4 ), that works on the portfolio HTML structure.

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -1,16 +1,32 @@
...
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-    </div>
-  </body>
-</html>
+
+<head>
+	<meta charset="UTF-8">
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/index.html]: [indent 2 spaces at a time](https://google.github.io/styleguide/htmlcssguide.html#Indentation)

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -1,16 +1,32 @@
...
+		<div class="section">
+		</div>
+        <div class="section" id="projects">
+            <a class="card">InfluencerAds</a>
+            <a class="card">Permutation Palette Picker</a>
+            <a class="card">Auction Search Tool</a>
+            <a class="card">Genesis Trust Volunteer Portal</a>
+            <a class="card">2048 Clone</a>
+            <a class="card">Something</a>
+		</div>
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/index.html]: the indentation of the closing `</div>` should match the indentation of the corresponding opening `<div>` tag, here and elsewhere.

File: portfolio/src/main/webapp/style.css [hunk 3]
```diff
@@ -0,0 +1,48 @@
+body{
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/style.css]: add a space before `{`, here and elsewhere

File: portfolio/src/main/webapp/style.css [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    margin:0;
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/style.css]: add a space after `:`, here and elsewhere where it is missing

File: portfolio/src/main/webapp/style.css [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+
+#fullpage{
+    min-width: 100vw;
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/style.css]: [alphabetize declarations](https://google.github.io/styleguide/htmlcssguide.html#Declaration_Order), here and elsewhere

File: portfolio/src/main/webapp/style.css [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    min-height: 100vh;
+    max-height: 100vh;
+    overflow: hidden;
+    min-width: 100%;
+    max-width: 100%;
+}
+
+#projects{
+    display: grid;
+    grid-template-rows: repeat(2,1fr);
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/style.css]: use [the `grid-template` shorthand property](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template) instead (reference: https://google.github.io/styleguide/htmlcssguide.html#Shorthand_Properties)

File: portfolio/src/main/webapp/style.css [hunk 7]
```diff
@@ -0,0 +1,48 @@
+body{
+    margin:0;
+}
+
+
+#fullpage{
+    min-width: 100vw;
+    min-height: 100vh;
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/style.css]: [Indent by 2 spaces at a time.](https://google.github.io/styleguide/htmlcssguide.html#Indentation)

File: portfolio/src/main/webapp/index.html [hunk 8]
```diff
@@ -7,10 +7,37 @@
...
   <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
+    <div id="fullpage">
+      <div class="section"></div>
+      <div class="section"></div>
+      <div class="section" id="projects">
+        <a class="card">
```
Comment (krasimirgg) [inline portfolio/src/main/webapp/index.html]: The [`<a>` tag](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) is usually associated with hyperlinks. Here it looks like it's used as a contained for two `<span>` tags and it's missing an `href` attribute. Either add an `href` attribute, or replace this with a more appropriate content tag like a `<div>`, here and below. 
Reference:
https://google.github.io/styleguide/htmlcssguide.html#Semantics

File: portfolio/src/main/webapp/style.css (hunk 7 above)
Comment (samuelsogbesan) [reply portfolio/src/main/webapp/style.css]: Spacing reformatted in commit 474248a

**Review by krasimirgg — APPROVED** [review]
LGTM

**Pull request merged**